app/tiebout.hoon

```diff
@@ -0,0 +1,253 @@
+::
+::::  /app/tiebout/hoon
+::
+/?  309
+/-  hall
+/+  tiebout
+=,  tiebout
+=,  eyre
+::
+::  state:
+::
+~%  %tiebout  ..^is  ~
+|_  [bol=bowl:gall sta=state]
+::
+::  +this: app core subject
+::
+++  this  .
+::
+::  +prep: set up app state, upgrade app state
+::
+++  prep
+  |=  old=(unit state)
+  ^-  (quip move _this)
+  ?~  old
+    :-  ~
+    %=  this
+      king.tie.sta  ~dopzod
+      baseurl.tie.sta  'https://api.push.apple.com/3/device/'
+    ==
+  ?-  -.u.old
+    %0
+      [~ this(sta u.old)]
+  ==
+::
+::  +coup: receive acknowledgement for poke, print error if it failed
+::
+++  coup
+  |=  [wir=wire err=(unit tang)]
+  ^-  (quip move _this)
+  ?~  err
+    [~ this]
+  (mean u.err)
+::
+::  +poke-noun: receive debugging actions
+::
+++  poke-noun
+  |=  act=action
+  ^-  (quip move _this)
+  (poke-tiebout-action act)
+::
+::  +poke-tiebout-action: main action handler
+::
+++  poke-tiebout-action
+  |=  act=action
+  ^-  (quip move _this)
+  ?-  -.act
+    $king
+      (set-king +.act)
+    $token
+      (set-token +.act)
+    $baseurl
+      (set-baseurl +.act)
+    $add-circle
+      (add-circle +.act)
+    $del-circle
+      (del-circle +.act)
+    $notify
+      (send-notify +.act)
+  ==
```

Comment:
Old syntax!

The indentation is backwards here, I'd just single-line each of these cases anyway. They're short enough.

Reply:
fixed